pkg/image/controller/controller_test.go

```diff
@@ -66,14 +68,15 @@ func (f *fakeDockerRegistryClient) ImageByID(namespace, name, id string) (*docke
 	return nil, dockerregistry.NewImageNotFoundError(fmt.Sprintf("%s/%s", namespace, name), id, "")
 }
 
-func TestControllerNoDockerRepo(t *testing.T) {
+func TestControllerNoOp(t *testing.T) {
 	cli, fake := &fakeDockerRegistryClient{}, &client.Fake{}
 	c := ImportController{client: cli, streams: fake, mappings: fake}
 
 	stream := api.ImageStream{
 		ObjectMeta: kapi.ObjectMeta{
-			Name:      "test",
-			Namespace: "other",
+			Annotations: map[string]string{api.DockerImageRepositoryCheckAnnotation: util.Now().UTC().Format(time.RFC3339)},
+			Name:        "test",
+			Namespace:   "other",
 		},
 	}
 	other := stream
```

Comment:
Yeah, I am worried with that as well. I'll fix it when addressing comments tomorrow.